Dockerfile

```
@@ -2,17 +2,20 @@ FROM python:3.9 as base
 RUN pip install poetry
 WORKDIR /app
 COPY poetry.lock pyproject.toml ./
-RUN poetry install
+RUN poetry config virtualenvs.create false --local && poetry install
 COPY todo_app todo_app
 COPY test test
+COPY entrypoint.sh entrypoint.sh
 
 FROM base as development
 ENTRYPOINT ["poetry","run","flask","run", "--host","0.0.0.0"]
 
+# Docker Testing
+FROM base as test
+ENTRYPOINT ["poetry", "run", "pytest"]
+
+#docker prod
 FROM base as production
 RUN poetry add gunicorn
-ENTRYPOINT ["poetry","run","gunicorn","todo_app.app:create_app()", "-b","0.0.0.0:80"]
+ENTRYPOINT ["sh","./entrypoint.sh"]
```

Since you switched to use an environment variable for your PORT argument as Heroku insists, following your Readme instructions no longer works out the box since locally we aren't passing a PORT environment variable. We could fix that by asking users to pass one, but probably nicer is to set a default value here so they don't need to think about it, but Heroku can still override the default:

```suggestion
ENV PORT=80
ENTRYPOINT ["sh","./entrypoint.sh"]
```